BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Gio/GioViewController.swift

```diff
@@ -1,19 +1,120 @@
-/*
- - UISegmentedControl
-     - How do you customize it.
-     - How do you add an action/target.
- - UISwitch (Gio)
-     - How do you customize it.
-     - How do you add an action/target.
- */
-
 import UIKit
 
 class GioViewController: UIViewController {
+    
+    // MARK: - Variables
+    let segmentedControlItems = ["Ada Lovelace", "Nikola Tesla", "Michael Scott"]
+    
+    let quotes = ["The intellectual, the moral, the religious seem to me all naturally bound up and interlinked together in one great and harmonious whole.",
+                  "What one man calls God, another calls the laws of physics.",
+                  "Sometimes I'll start a sentence and I don't even know where it's going. I just hope I find it along the way.",
+                  "CHINA!"]
+    
+    private lazy var segmentedControl: UISegmentedControl = {
+        let segmentedControl = UISegmentedControl(items: segmentedControlItems)
+        
+        /* uncomment 17-28 */
+//        let font = UIFont(name: "Verdana", size: UIFont.systemFontSize)!
+//        segmentedControl.setTitleTextAttributes([
+//            NSAttributedString.Key.font: font,
+//            NSAttributedString.Key.foregroundColor: UIColor.lightGray
+//            ], for: .normal)
+//
+//        segmentedControl.setTitleTextAttributes([
+//            NSAttributedString.Key.font : font,
+//            NSAttributedString.Key.foregroundColor: UIColor.darkGray
+//            ], for: .selected)
+//
+//        segmentedControl.tintColor = .clear
 
+        //segmentedControl.backgroundColor = .gray
+        
+        //segmentedControl.insertSegment(withTitle: "Donald Trump", at: segmentedControl.numberOfSegments + 1, animated: true)
+        //segmentedControl.setTitle(String(segmentedControlItems[0].reversed()), forSegmentAt: 0) /* segment must already exist */
```

Comment:
Is this dead code or to demo during the meetup?

Reply:
It's going to be used to demo during the meetup